nodeup/pkg/model/encryption.go

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package model
+
+import (
+	"fmt"
+
+	"k8s.io/apiserver/pkg/server/options/encryptionconfig"
```

Comment:
I think a general convention ... std lib, local imports and then external .. Could we drop the encryptionconfig import down

Reply:
Happy to do this, but it doesn't seem to be consistent with the rest of the repo. The convention seems to be: std lib, k8s.io libraries, external libraries e.g. in `nodeup/pkg/model/kubeapiserver.go`, `nodeup/pkg/model/kubecontrollermanager.go` etc.

Reply:
The imports formatting is not consistent through the project ;( But yes @gambol99 has it correct